docs/connectObject.md

```diff
@@ -0,0 +1,258 @@
+# Connect Object class
+
```

Comment:
can we move this to the Readme or code, not sure we need an individual markdown file, was there a reason to create separate documentation around this

Reply:
As you can see the ConnectObject Instance is actually a module by itself providing way to connect to the AEP API without the need of other modules, it can also serve as a way to rotate your client secret programmatically. 
As a specific class and method available, it has to follow the current structure and get its own markdown file. 

The readMe would be pretty long with that.